selfdrive/ui/onroad/augmented_road_view.py

```diff
@@ -94,6 +97,12 @@ def render(self, rect):
     # End clipping region
     rl.end_scissor_mode()
 
+    # Handle click events if no HUD interaction occurred
+    if not self._hud_renderer.handle_mouse_event():
+      if self.on_click and rl.is_mouse_button_pressed(rl.MouseButton.MOUSE_BUTTON_LEFT):
+        if rl.check_collision_point_rec(rl.get_mouse_position(), self._content_rect):
+          self.on_click()
+
```

Comment:
move to function that handles mouse clicks to prepare for base class?

Reply:
think we can handle this better after https://github.com/commaai/openpilot/pull/35489